src/Core/Layer/GeometryLayer.js

```diff
@@ -0,0 +1,104 @@
+import { EventDispatcher } from 'three';
+import Picking from '../Picking';
+
+function GeometryLayer(id, object3d) {
```

Comment:
you could use `class`?

Reply:
logically, it should be called meshLayer

Reply:
> you could use `class`?

Wouldn't that be losing coherence with the actual code or do you wish to push the use of `class` elsewhere ?

> logically, it should be called meshLayer

But then the `type` would be `mesh`, and I find it too specific.